### PR TITLE
beancount: add myself to maintainers

### PR DESCRIPTION
**Pull request by bhipple**
Saw it's unmaintained; since I use this regularly, happy to help maintain. Also
fixing a broken URL.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/beancount/default.nix [hunk 1]
```diff
@@ -42,7 +54,6 @@ buildPythonPackage rec {
         generate a variety of reports from them, and provides a web interface.
     '';
     license = licenses.gpl2;
```
Comment (fabaff) [inline pkgs/development/python-modules/beancount/default.nix:56]: Could you please update the license in the process?  It's [`gpl2Only`](https://github.com/beancount/beancount#copyright-and-license) as `gpl2` is obsolete.

Thanks

Comment (Mic92) [inline pkgs/development/python-modules/beancount/default.nix:56]:
```suggestion
    license = licenses.gpl2Only;
```

Comment (Mic92) [reply pkgs/development/python-modules/beancount/default.nix]: done.

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by blaggacao** [comment]
@bhipple 

> I use this regularly

Thank you for inspiration occasioned by this statement: it looks ideal for keeping records of a small business. Docs are a bit wordy and convoluted (from an accountants perspective), but well.

Do you use [pinto](https://pypi.org/project/pinto/)? Would you be interested in packaging it for nixpkgs?

I've also seen quite an ecosystem of contributions to beancount, so I immediatly started to wonder if / how we coukd reflect that and make that accessible to nixpkgs.

**Comment by bhipple** [comment]
I love plain text accounting and find the whole idea wonderful.  I'm relatively new in converting to beancount and haven't explored much beyond the basics, but I've been using ledger-cli (the original C++ implementation) for almost 5 years.  If I end up using pinto I'll package it.  

If you're looking to learn more, check out:
https://plaintextaccounting.org/
